### PR TITLE
translation: Update chap_tree/summary.md

### PR DESCRIPTION
**Pull request by steventimes**
If this pull request (PR) pertains to **Chinese-to-English translation**, please confirm that you have read the contribution guidelines and complete the checklist below:

- [*] This PR represents the translation of a single, complete document, or contains only bug fixes.
- [*] The translation accurately conveys the original meaning and intent of the Chinese version. If deviations exist, I have provided explanatory comments to clarify the reasons.

If this pull request (PR) is associated with **coding or code transpilation**, please attach the relevant console outputs to the PR and complete the following checklist:

- [ ] I have thoroughly reviewed the code, focusing on its formatting, comments, indentation, and file headers.
- [ ] I have confirmed that the code execution outputs are consistent with those produced by the reference code (Python or Java).
- [ ] The code is designed to be compatible on standard operating systems, including Windows, macOS, and Ubuntu.

### CODE REVIEW

**Comment by steventimes** [comment]
I have changed some wording, and for line 37, since I do not know c++, I am not sure whether to use "function" or "method".

**Comment by yuelinxin** [comment]
Functions defined in an object in cpp is usually referred as a “method”, this follows the terminology commonly accepted in object-oriented programming.

File: en/docs/chapter_tree/summary.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 # Summary
 
-### Key review
+### Key Highlights
```
Comment (yuelinxin) [inline en/docs/chapter_tree/summary.md:3]: we have fixed this title throughout this book so no need to change this

**Review by yuelinxin — COMMENTED** [review]
Great work! Please consider my comment on the title and also the function terminology.

**Comment by steventimes** [comment]
I have updated the file with the changed title, and named back to "method". Thank you

**Review by yuelinxin — APPROVED** [review]
Great work, thank you!

File: en/docs/chapter_tree/summary.md [hunk 2]
```diff
@@ -3,24 +3,24 @@
 ### Key review
 
 - A binary tree is a non-linear data structure that reflects the "divide and conquer" logic of splitting one into two. Each binary tree node contains a value and two pointers, which point to its left and right child nodes, respectively.
-- For a node in a binary tree, the tree formed by its left (right) child node and all nodes under it is called the node's left (right) subtree.
-- Related terminology of binary trees includes root node, leaf node, level, degree, edge, height, and depth, among others.
+- For a node in a binary tree, its left (right) child node and the tree formed below it is called the node's left (right) subtree.
```
Comment (magentaqin) [inline en/docs/chapter_tree/summary.md:6]: "is called" => "are collectively called"

File: en/docs/chapter_tree/summary.md [hunk 3]
```diff
@@ -3,24 +3,24 @@
...
 - Pre-order, in-order, and post-order traversals are all depth-first search methods, reflecting the traversal manner of "going to the end first, then backtracking to continue." They are usually implemented using recursion.
 - A binary search tree is an efficient data structure for element searching, with the time complexity of search, insert, and remove operations all being $O(\log n)$. When a binary search tree degrades into a linked list, these time complexities deteriorate to $O(n)$.
 - An AVL tree, also known as a balanced binary search tree, ensures that the tree remains balanced after continuous node insertions and removals through rotation operations.
-- Rotation operations in an AVL tree include right rotation, left rotation, right-then-left rotation, and left-then-right rotation. After inserting or removing nodes, an AVL tree performs rotation operations from bottom to top to rebalance the tree.
+- Rotation operations in an AVL tree include right rotation, left rotation, right-left rotation, and left-right rotation. After inserting or removing nodes, the AVL tree performs rotation operations from bottom to top to rebalance the tree.
```
Comment (magentaqin) [inline en/docs/chapter_tree/summary.md:15]: If you use "inserting/removing", the subject should be "we". How about updating this to "After node insertion or removal, the AVL tree rebalances itself by performing these rotations in a bottom-up manner."?

File: en/docs/chapter_tree/summary.md [hunk 4]
```diff
@@ -3,24 +3,24 @@
...
 
 ### Q & A
 
 **Q**: For a binary tree with only one node, are both the height of the tree and the depth of the root node $0$?
 
 Yes, because height and depth are typically defined as "the number of edges passed."
 
-**Q**: The insertion and removal in a binary tree are generally completed by a set of operations. What does "a set of operations" refer to here? Can it be understood as the release of resources of the child nodes?
+**Q**: The insertion and removal in a binary tree are generally accomplished by a set of operations. What does "a set of operations" refer to here? Can it be understood as the release of resources of the child nodes?
```
Comment (magentaqin) [inline en/docs/chapter_tree/summary.md:23]: Can it be understood...? => Does it imply releasing the resources of the child nodes?

**Comment by steventimes** [comment]
changes pushed according to your advise, thank you so much! @magentaqin

**Review by krahets — APPROVED** [review]

**Pull request merged**